scripts/subtitle_search_eval.py

```
@@ -0,0 +1,546 @@
+        base = p.rsplit('/', 1)[-1]
+        if base.startswith('.'):
+            continue
+        low = base.lower()
+        if not any(low.endswith(e) for e in VIDEO_EXT):
+            continue
+        if any(low.endswith(b) for b in BAD_SUFFIX):
+            continue
+        videos.append(p)
+
+    print(f"提取视频文件 {len(videos)} 个\n")
+    ok = 0; bad = 0; skip = 0
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)